src/views/workflow-history/workflow-history-events-card/workflow-history-events-card.tsx

```diff
@@ -0,0 +1,71 @@
+'use client';
+import React from 'react';
+
+import { Accordion, Panel } from 'baseui/accordion';
+import { Skeleton } from 'baseui/skeleton';
+
+import useStyletronClasses from '@/hooks/use-styletron-classes';
+
+import getBadgeContainerSize from '../workflow-history-event-status-badge/helpers/get-badge-container-size';
+import WorkflowHistoryEventStatusBadge from '../workflow-history-event-status-badge/workflow-history-event-status-badge';
+
+import { cssStyles, overrides } from './workflow-history-events-card.styles';
+import { type Props } from './workflow-history-events-card.types';
+
+export default function WorkflowHistoryEventsCard({
+  eventsMetadata,
+  showMissingEventPlaceholder,
+}: Props) {
+  const { cls, theme } = useStyletronClasses(cssStyles);
+
+  if (!eventsMetadata?.length && !showMissingEventPlaceholder) return null;
+  return (
+    <Accordion overrides={overrides.accordion} accordion>
```

Comment:
Do we want only one panel to be open at a time? If not, "accordion" needs to be set to false.

Reply:
It feels more compact doing and reduces scrolling, So yes lets keep it this way unless proven otherwise